modules/orca/variables.tf

```diff
@@ -4,6 +4,7 @@
 variable "aws_profile" {
   type        = string
   description = "AWS profile used to deploy the terraform application."
+  default = null
```

Comment:
Could also have the default be `default` but this does communicate clearly to Cumulus that they can use 'null' again. Thoughts?